README.md

```diff
@@ -37,8 +37,9 @@ $ git clone https://github.com/SAP/odfuzz && cd odfuzz
         ```
         2. Run the container:
         ```
-        $ sudo docker run --dns=10.17.121.71 -ti odfuzz:1.0
+        $ sudo docker run --dns=10.17.121.71 -v <absolute_hostpath>:/odfuzz odfuzz:1.0 <arguments>
```

Comment:
This will hide contents for the image shipped directory /odfuzz - not sure you want it.

Reply:
Yes, this isn't working as intended. It should be a different path, where the results are stored.